src/threads.js

```diff
@@ -2706,14 +2706,14 @@ Process.prototype.reportLetter = function (idx, string) {
     if (string instanceof List) { // catch a common user error
         return '';
     }
+    str = isNil(string) ? '' : string.toString();
```

Comment:
we probably want to declare this as a local variable (```var str 0 ...```), so it doesn't accidentally become global...

Reply:
I think github cuts it off from the view, but the bar is rare in the existing definition. :)

Reply:
wow, you're right. That's funny!